package.json

```diff
@@ -8,7 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^27.5.2",
     "@types/node": "^16.18.38",
-    "@types/react": "^18.2.14",
+    "@types/react": "^18.2.15",
```

Comment:
The vscode workspace was complaining about types that I was able to resolve by adding this dependency